accelerometer/summariseEpoch.py

```diff
@@ -7,13 +7,16 @@
 import pandas as pd
 import pytz
 import sys
-
+import scipy as sp
```

Comment:
Add this to list of install requirements in setup.py (~line 20)

Reply:
resolved in 6d4af93faddf9b831dcdd07690b885da862b6e18